### ISSUE TITLE
Color on Search File / Go to File Highlight

### ISSUE THREAD

**Issue by chrysillala** [prometheux-ar/cyberpunk#89]
I'm loving your Activate UMBRA Protocol theme, but I find that when I search for file and navigating with arrow there's no color to show my selected file.
![image](https://user-images.githubusercontent.com/4686015/81137543-5e15cd00-8f89-11ea-973c-7d5c651a8c0c.png)

Though when I hover with mouse, I can see the text turned green. And I believe most of us would prefer navigating with arrow over mouse 😄 
![image](https://user-images.githubusercontent.com/4686015/81137972-97027180-8f8a-11ea-93df-d575b1a1e773.png)

It would be nice if we could have the same behaviour when navigating with arrow or mouse.
Thanks!

**Comment by ayoubsousali1** [comment]
i have the same problem

**Comment by brett-wilcox** [comment]
Yeah, this is definitely an issue. I will see if I can track down the color value and do a pull request.

**Comment by anthonyholmes** [comment]
You can override properties via settings. I'm using this for now

```
"workbench.colorCustomizations": {
    "[Activate UMBRA protocol]": {
        "quickInput.background": "#100D23",
    }
},
```
